packages/@react-stately/combobox/src/useComboBoxState.ts

```diff
@@ -118,6 +118,7 @@ export function useComboBoxState<T extends object>(props: ComboBoxStateOptions<T
 
       menuOpenTrigger.current = trigger;
       triggerState.open(focusStrategy);
+      selectionManager.setFocused(true);
```

Comment:
With the change to this [line](https://github.com/adobe/react-spectrum/blob/f345629a6e415745b84d893ea425a7ac0f7a968b/packages/%40react-aria/selection/src/useSelectableItem.ts#L317), a selectable item is only considered focused if the collection itself is focused as well. This gets set on option [hover](https://github.com/adobe/react-spectrum/blob/85fcd56cd1c0eb19bd14e59206416f6b77987e7b/packages/%40react-aria/listbox/src/useOption.ts#L143), collection [focus](https://github.com/adobe/react-spectrum/blob/85fcd56cd1c0eb19bd14e59206416f6b77987e7b/packages/%40react-aria/selection/src/useSelectableCollection.ts#L285), or on collection mount if [autofocus is true](https://github.com/adobe/react-spectrum/blob/85fcd56cd1c0eb19bd14e59206416f6b77987e7b/packages/%40react-aria/selection/src/useSelectableCollection.ts#L347). For a collection that has `shouldUseVirtualFocus=true`, we rely on the autofocus being true, set for ComboBox [via this line](https://github.com/adobe/react-spectrum/blob/46a729bce2761ace4f4ed83552ed024120880e6e/packages/%40react-spectrum/combobox/src/ComboBox.tsx#L169) and comes from calling [open with various focus strategies](https://github.com/adobe/react-spectrum/blob/46a729bce2761ace4f4ed83552ed024120880e6e/packages/%40react-stately/combobox/src/useComboBoxState.ts#L120). The problem is that the combobox's menu could be initially opened without a focusStrategy (aka via mouse click or clearing the field) which means we don't have a autoFocus value of true, thus making it so the menu's `isFocused` state is never true until we hover an option

Reply:
I'm a bit torn on if we should consider the combobox menu as 'isFocused` on open, or if it should only be considered focused when using the up/down arrow keys (aka putting this logic in useComboBox's onKeydown). Also, technically we should set `isFocused` to false when the menu closes for correctness...

Reply:
what from hovering makes it true? and why can't we do that with the keyboard?

Reply:
for posterity: discussed this with @snowystinger and @reidbarber, decided to keep this logic in the combobox space for now instead of the collection level hooks since we don't have any other `shouldUseVirtualFocus` cases in Spectrum for now. Keeping it in useComboBoxState since the clearing of the tracked selectedKey happens there already and we are guarded against other edge cases via keyboard modality checks elsewhere.